actors/builtin/verifreg/verified_registry_actor.go

```diff
@@ -262,6 +264,7 @@ func (a Actor) RestoreBytes(rt vmr.Runtime, params *RestoreBytesParams) *adt.Emp
 		builtin.RequireNoErr(rt, err, exitcode.ErrIllegalState, "failed to load verifiers")
 
 		// validate we are NOT attempting to do this for a verifier
+		// XXX: This seems expensive and pointless?
```

Comment:
This is going to cost us some (expensive) state accesses, to check something that should be impossible. We can keep it, it just doesn't seem worth it.

Reply:
Should be impossible because a verifier should not appear in the clients table? Yes, SGTM, just leave a comment noting that.

Reply:
Not quite because we may have removed it from the clients table (if zero).

However, this method can only be called by cron when a verified deal times out.

Reply:
Ok, I'm reverting my change here unless we resolve https://github.com/filecoin-project/specs-actors/pull/758/files#r470369407.